### DISCUSSION TITLE
do you have plan to combine the tesseract_ros with moveit

### DISCUSSION THREAD

**Discussion by dbdxnuliba** [tesseract-robotics/tesseract_ros#186] [Ideas]
do you have plan to combine the tesseract_ros with moveit

**Comment by marip8** [comment]
The short answer is that no plan currently exists. MoveIt and Tesseract are conceptually similar in that they are both frameworks for motion planning for robotic manipulators, but they have different focus areas and some different features/architectures. At this point, there are several capabilities built into Tesseract that are fundamentally not possible in MoveIt currently (@Levi-Armstrong can probably enumerate them if necessary), and many of those were part of the reason that Tesseract was created in the first place. It would be tough to combine Tesseract with MoveIt as it stands now, but we definitely share and adopt good new features in both directions as they come up.

**Reply by dbdxnuliba** [reply]
OK, I am  looking forward the combined work
